geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/GeolocationPlugin.java

```diff
@@ -0,0 +1,201 @@
+package com.capacitorjs.plugins.geolocation;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+import com.getcapacitor.annotation.CapacitorPlugin;
+import com.getcapacitor.annotation.Permission;
+import java.util.HashMap;
+import java.util.Map;
+
+@CapacitorPlugin(
+    name = "Geolocation",
+    permissions = {
+        @Permission(permission = Manifest.permission.ACCESS_COARSE_LOCATION, alias = "location"),
+        @Permission(permission = Manifest.permission.ACCESS_FINE_LOCATION, alias = "location")
```

Comment:
Why do these share an alias? App devs may want to differentiate between the two, right?

Reply:
Yes they might but as the plugin is currently written, these two permissions were both requested together. The plugins currently dont offer a way to differentiate between wither the developer wants to use coarse or fine location (in iOS or Android). If we want to look at allowing that we might want to plan it as future work. There are also changes to background location coming with Android 11 that need to be addressed in the future, potentially.